osu.Game/Rulesets/Mods/ICanBeToggledDuringReplay.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Bindables;
+
+namespace osu.Game.Rulesets.Mods
+{
+    public interface ICanBeToggledDuringReplay : IApplicableMod
+    {
+        Bindable<bool> ReplayLoaded
+        {
+            get;
+        }
+
+        bool IsDisable
+        {
+            get;
+        }
+
+        void OnToggle();
```

Comment:
* No xmldoc, no description of what the interface is for or what the individual members do. This is not acceptable for a ruleset API.
* Why is `ReplayLoaded` in this interface? Ideally it would not be.
* What is `IsDisable` even supposed to be?